### PR TITLE
Adds a battery status demo

### PR DESCRIPTION
**Pull request by PaulKinlan**
It demonstrates the core principles of the API, the values and also the events.

### CODE REVIEW

File: battery-status/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Battery Status Sample
+=====================
+
+See https://googlechrome.github.io/samples/battery-status for a live demo.
+
+Learn more at https://www.chromestatus.com/features/4537134732017664
```
Comment (jeffposnick) [inline battery-status/README.md]: Can you link to https://www.chromestatus.com/feature/4537134732017664 ("feature", not "features") to go directly to the landing page?

File: battery-status/index.html [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+    <meta name="theme-color" content="#3372DF">
+
+    <link rel="icon" href="../images/favicon.ico">
+
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+  <body>
+    <h1>Battery Status Sample</h1>
+    <p>Available in <a href="https://www.chromestatus.com/features/4537134732017664">Chrome 38</a></p>
```
Comment (jeffposnick) [inline battery-status/index.html]: features → feature

**Comment by PaulKinlan** [comment]
Changed the spellings.

**Comment by jeffposnick** [comment]
Thanks!

**Pull request merged**